modules/nf-core/fgbio/callduplexconsensusreads/meta.yml

```diff
@@ -10,18 +10,23 @@ tools:
       homepage: http://fulcrumgenomics.github.io/fgbio/
       documentation: http://fulcrumgenomics.github.io/fgbio/tools/latest/CallDuplexConsensusReads.html
       tool_dev_url: https://github.com/fulcrumgenomics/fgbio
-      licence: "['MIT']"
+      licence: ["MIT"]
 input:
-  # Only when we have meta
   - meta:
       type: map
       description: |
         Groovy Map containing sample information
         e.g. [ id:'test', single_end:false ]
-  - bam:
+  - grouped_bam:
```

Comment:
The tuple is flattened in this list.
It does need to be consistent between the main.nf and meta.yml though. I changed them all to be `bam` for consistency (although them being grouped_bam does make it clear what the input needs to be from the code rather than the meta.yml)